### ISSUE TITLE
[Feature request] yo yeogurt:page [PAGE_NAME]

### ISSUE THREAD

**Issue by oninross** [larsonjj/generator-yeogurt#200]
When running the command, have the option of placing the generated pages in a folder

**Comment by larsonjj** [comment]
Hi @oninross,

This possible using the page subgenerator like so:

```
yo yeogurt:page about/company
```

Which will create a file in the following location

```
create src/about/company/index.nunjucks
```

NOTE: You can specify subfolders by adding `/` to the command

Let me know if that helps

**Comment by nik-s** [comment]
It would be great if `- var baseUrl = '../'` would be updated to `'../../'` when you generate a page in a subfolder. Is this possible? Thanks.

**Comment by oninross** [comment]
hi @larsonjj,

I tried the said command (ex. `yo yeogurt:page pages/standard`) but it generated a folder named pagesstandard instead.  This was the one I experienced last itme

**Comment by larsonjj** [comment]
Fixed in #226
